browser/ui/brave_browser.cc

```diff
@@ -200,6 +204,36 @@ void BraveBrowser::TabCustomTitleChanged(
   }
 }
 
+content::WebContents* BraveBrowser::AddNewContents(
+    content::WebContents* source,
+    std::unique_ptr<content::WebContents> new_contents,
+    const GURL& target_url,
+    WindowOpenDisposition disposition,
+    const blink::mojom::WindowFeatures& window_features,
+    bool user_gesture,
+    bool* was_blocked) {
+  // Only handle NEW_FOREGROUND_TAB disposition (target="_blank" links)
+  if (disposition == WindowOpenDisposition::NEW_FOREGROUND_TAB &&
+      user_gesture && source && !target_url.is_empty()) {
```

Comment:
Q. I'm wondering why we check `user_gesture` and when it could be false. If `user_gesture` is false when `window.open` method, should we handle that case here too?

Reply:
> Maybe we want to add more cases to route to right pane. but started with only this case in this PR.

Thanks, definitely okay with follow ups. I was wondering what we'd like to do with a case where `window.open` opens a new tab. like, https://codepen.io/sangwoo108/pen/EaKdJoz